cluster-provision/images/vm-image-builder/alpine-cloud-init/create-image.sh

```diff
@@ -1,18 +1,20 @@
 #!/bin/sh
 set -ex
 
+CRI=${CRI:-docker}
```

Comment:
here forgotten right ?
also please update the readme,
no need to export and set CRI_BIN (appears twice)

Reply:
Done